torch/csrc/fx/node.cpp

```diff
@@ -0,0 +1,269 @@
+#include <torch/csrc/fx/node.h>
+
+#include <structmember.h>
+
+////////////////////////////////
+// NodeBase
+///////////////////////////////
+
+struct NodeBase {
+  PyObject_HEAD bool _erased;
+  NodeBase* _prev;
+  NodeBase* _next;
+};
+
+static PyObject* NodeBase_new(
```

Comment:
Isn't that the default implementation? I would expect you don't need to provide it if you don't have special handling.

Reply:
If I remove this implementation, I get
```
TypeError: cannot create 'Node' instances
```

Reply:
Ho right they don't inherit it. Ok!